gravitee-apim-rest-api/gravitee-apim-rest-api-management-v4/gravitee-apim-rest-api-management-v4-rest/src/main/resources/management-openapi-v4.yaml

```diff
@@ -599,7 +599,8 @@ components:
                     example: v1.0
                     minLength: 1
                 definitionVersion:
-                    $ref: "#/components/schemas/DefinitionVersion"
```

Comment:
why we not keep this ref ?

Reply:
We will only have "v4" for this module. So we don't need the enum DefinitionVersion.